app/config.py

```diff
@@ -3,6 +3,6 @@
 
 class Config(object):
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'you-will-never-guess'
-    SQALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
```

Comment:
I fixed this area in the Login_form branch

Reply:
Here is what it should be: 
` SQLALCHEMY_DATABASE_URI = 'sqlite:///databases/app.db'`